src/style.css

```diff
@@ -0,0 +1,101 @@
+* {
+  margin: 0;
```

Comment:
don't do that, use * only for box-sizing, other properties too heavy for this selector